src/plugins/workspace/public/components/workspace_form/workspace_form.tsx

```diff
@@ -32,27 +34,36 @@ export const WorkspaceForm = (props: WorkspaceFormProps) => {
     operationType,
     permissionEnabled,
     workspaceConfigurableApps,
-    permissionLastAdminItemDeletable,
   } = props;
   const {
     formId,
     formData,
     formErrors,
     numberOfErrors,
+    numberOfChanges,
     handleFormSubmit,
-    handleColorChange,
     handleUseCasesChange,
     handleNameInputChange,
     setPermissionSettings,
     setSelectedDataSources,
     handleDescriptionChange,
   } = useWorkspaceForm(props);
   const workspaceDetailsTitle = i18n.translate('workspace.form.workspaceDetails.title', {
-    defaultMessage: 'Enter Details',
+    defaultMessage: 'Enter details',
   });
+  const disabledUserOrGroupInputIdsRef = useRef(
```

Comment:
Nit: should it use `useMemo` so that the computed id list will update when `defaultValues.permissionSettings` changed?

Reply:
This is by intentionally. The `defaultValues` should not be changed after workspace form component mount. It's not reflect to the form values. It's not reflect to the latest form values. We need to use these default values to disable the user or group selector .